src/crawlee/configuration.py

```diff
@@ -229,7 +243,7 @@ class Configuration(BaseSettings):
             )
         ),
     ] = False
-    """This setting is currently unused. For more details, see https://github.com/apify/crawlee-python/issues/670."""
+    """This setting is currently unused."""
```

Comment:
I think it's safe to just drop this and never look back.

Reply:
same as the other one